lms/envs/aws.py

```diff
@@ -119,8 +121,8 @@
     """
 
     'collect_stats': {
-    'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
-    'schedule': 30, # number in seconds.
+        'task': 'openedx.core.djangoapps.edx_global_analytics.tasks.collect_stats',
+        'schedule': crontab(hour=0, minute=random.randint(1, 59))
```

Comment:
One last thing. @idegtiarov has told me it's better to move schedule declaration into a decorator to wrap task function.
@DmytryiStriletskyi please ask him ho easy is to do that.